crates/kubelet/src/config.rs

```diff
@@ -62,12 +71,20 @@ impl Config {
             default_node_ip(&mut hostname.clone(), &addr)
                 .expect("unable to get default node IP address")
         });
+        let pfx_path = opts
+            .pfx_path
+            .unwrap_or_else(|| PathBuf::from("./config/certificate.pfx"));
```

Comment:
Not something that needs to be addressed now, but we should probably define a common path (like `~/.krustlet`) that things like certs and pod logs can go into by default